leshan-server-core/src/main/java/org/eclipse/leshan/server/observation/ObservationListener.java

```diff
@@ -30,5 +30,13 @@
      * @param observation the observation for which new data are received
      * @param reponse the lwm2m response received
      */
-    void newValue(Observation observation, ObserveResponse response);
+    void onResponse(Observation observation, ObserveResponse response);
```

Comment:
Just wondering what happened to `cancelled` method here. How will the listener gets notified about an observation cancellation?